cpg_pipes/images.py

```diff
@@ -0,0 +1,5 @@
+"""
+Docker images used in the pipelines.
+"""
+AR_REPO = 'australia-southeast1-docker.pkg.dev/cpg-common/images'
```

Comment:
Would common image reference location replace the alternative cpg_pipes.resources in [this branch](https://github.com/populationgenomics/production-pipelines/blob/initial_with_vep/cpg_pipes/resources.py)? If so, can we add a bunch more of the default container names in this MR?

Reply:
That's right, it's the same common image reference location!

What is `MR`? 

We can definitely add more images into the `cpg-common/images` repositories, and list them in the `cpg_pipes/images.py` if they are used in the pipeline (e.g. I will be adding more images while I'm moving more implemented jobs from the `initial` branch).

Reply:
Sorry, MR = merge request = pull request, I think that's just a terminology difference between gitlab and github

Reply:
Matt, answering your question:
> If so, can we add a bunch more of the default container names in this MR?
I wanted to try to stick to making small PRs, so I removed everything but one image from this module. I'm happy to add more images in a following PR!